hudi-flink-datasource/hudi-flink/src/main/java/org/apache/hudi/table/HoodieTableFactory.java

```diff
@@ -104,6 +106,7 @@ public DynamicTableSink createDynamicTableSink(Context context) {
     sanityCheck(conf, schema);
     setupConfOptions(conf, context.getObjectIdentifier(), context.getCatalogTable(), schema);
     setupSortOptions(conf, context.getConfiguration());
+    OptionUtils.printOptions(factoryIdentifier(), conf.toMap());
```

Comment:
Where can we see these logs, guess it's the submit client?

Reply:
en it will print on jobmanager's log